### ISSUE TITLE
breaking changes in NuGet.Protocol package version 4.9

### ISSUE THREAD

**Issue by AnthonySteele** [NuGet/Home#7593]
When updating the package [NuGet.Protocol to 4.9.1](https://www.nuget.org/packages/NuGet.Protocol/4.9.1) from 4.8, there are a few things that break. I am not aware of any docs on how to fix.

Particularly  I see the property `PackageSource.Origin`  has gone. 
And `NuGet.Configuration.Settings.ISettings` no longer has the property `Priority`.

This fails at compile-time after the minor version update.

**Comment by nkolev92** [comment]
//cc @PatoBeltran would have context.

**Comment by AnthonySteele** [comment]
Is this code equivalent? https://github.com/NuKeeperDotNet/NuKeeper/pull/587

**Comment by nkolev92** [comment]
@AnthonySteele It looks fine to me, but @PatoBeltran would know for sure if the order is correct there. (Not sure if it's relevant for your case however)

**Comment by PatoBeltran** [comment]
Hey @AnthonySteele, sorry that we broke you with these changes. While improving the `NuGet.Configuration` we tried to change as little as possible, but unfortunately `Priority` is one of the things that did change.

The change you point out in your PR is the perfect solution. The only difference between your previous code and this one is that we do a `Path.GetFullPath(...)` on the combined path (https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs#L584)
